torch/nn/quantized/modules/linear.py

```diff
@@ -30,13 +47,21 @@ def _save_to_state_dict(self, destination, prefix, keep_vars):
         (w, b) = self._weight_bias()
         destination[prefix + 'weight'] = w
         destination[prefix + 'bias'] = b
+        destination[prefix + 'dtype'] = self.dtype
 
     def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
                               missing_keys, unexpected_keys, error_msgs):
         self.set_weight_bias(state_dict[prefix + 'weight'], state_dict[prefix + 'bias'])
         state_dict.pop(prefix + 'weight')
         state_dict.pop(prefix + 'bias')
 
+        version = local_metadata.get('version', None)
```

Comment:
Do we need to serialize dtype? This is an attribute that is defined in the code at init.

Reply:
Yes. `dtype` is branched on in `set_weight_bias` which is called from the deserialization methods